### ISSUE TITLE
Graal Update package

### ISSUE THREAD

**Issue by nhh** [abertschi/graalphp#78]
Hi,

is there a way to add the graalphp as gu package?

As of today, it seems that there is no package available:

```
Downloading: Component catalog from www.graalvm.org
ComponentId              Version             Component name      Origin
--------------------------------------------------------------------------------
llvm-toolchain           20.2.0              LLVM.org toolchain  github.com
native-image             20.2.0              Native Image        github.com
python                   20.2.0              Graal.Python        github.com
R                        20.2.0              FastR               github.com
ruby                     20.2.0              TruffleRuby         github.com
wasm                     20.2.0              GraalWasm           github.com
```

I want to ".eval" some php today with the polyglot api, can we manually achieve this?

Thank you!

**Comment by abertschi** [comment]
Language compatibility capabilities are only partially implemented (#60).

You can install the language implementation as done in the CI sever:
https://github.com/abertschi/graalphp/blob/29e22de14d919ccd5647603b54ccd9eb25725d50/.travis.yml#L30

**Comment by nhh** [comment]
Thank you Andrin!
